Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -163,6 +163,21 @@ directory :file:`_assets/`.
     :bash:`composer dumpautoload` and the :file:`Resources/Public/` folder for
     that extension is symlinked to :file:`_assets/`.
 
+..  warning::
+    The :file:`_assets/` directory is not meant be manually changed. Also, it
+    is important for local development that all subdirectories are symlinks
+    to the specific Composer packages. Do not synchronize this directory
+    from a production instance back to your development instance (only the other
```

Comment:
why only the other way round? Also when this technically is possible, we should give the recommendation: don't sync assets folder, don't matter which direction.

Reply:
For deployment, you usually upload all `_assets` from your workspace to the production server, so that is the recommended direction. This is needed, without synching _assets there, the deployment is not useful (it would require executing composer on the remote machine).